web/packages/extension/src/utils.ts

```diff
@@ -1,6 +1,8 @@
 import type { Options } from "./common";
+import { DEFAULT_CONFIG as CORE_DEFAULT_CONFIG } from "ruffle-core";
 
 const DEFAULT_OPTIONS: Options = {
+    ...CORE_DEFAULT_CONFIG,
```

Comment:
Great catch (my bad...)! Now I think `DEFAULT_OPTIONS` can be made `Required<Options>`.

Reply:
Your changes fixed the previous duplication of initialization with default values, which can be a source of bugs and confusion, and therefore it is really nice that your changes fixed that duplication. I discovered (before your changes) that duplication, when I found out I had to update the initialization of a setting with a default value in multiple places.

I have added `Required<Options>`, and it did indeed compile and type-check. I have also rebased.